_data/operations.yml

```diff
@@ -1,138 +1,154 @@
+- affiliation:
+    name: University of Florida, Gainseville, FL
+    ror: 02y3ad647
   country: USA
   github: hoganwr
   groups:
   - editorial
   name: Bill Hogan
   orcid: 0000-0002-9881-1017
-  ror: 02y3ad647
   wikidata: Q56590169
-- affiliation: La Jolla Institute for Immunology, La Jolla, CA
+- affiliation:
+    name: La Jolla Institute for Immunology, La Jolla, CA
+    ror: 05vkpd318
   country: USA
   github: bpeters42
   groups:
   - editorial
   link: https://www.lji.org/labs/peters/
   name: Bjoern Peters
   orcid: 0000-0002-8457-6693
-  ror: 05vkpd318
   wikidata: Q60541156
-- affiliation: Lawrence Berkeley National Laboratory, Berkeley, CA
+- affiliation:
+    name: Lawrence Berkeley National Laboratory, Berkeley, CA
+    ror: 02jbv0t02
   country: USA
   github: cmungall
   groups:
   - technical
   link: https://github.com/cmungall/
   name: Chris Mungall
   orcid: 0000-0002-6601-2165
-  ror: 02jbv0t02
   wikidata: Q20746118
-- affiliation: University of Pennsylvania, Philadelphia, PA
+- affiliation:
+    name: University of Pennsylvania, Philadelphia, PA
+    ror: 00b30xv10
   country: USA
   github: cstoeckert
   groups:
   - outreach
   - editorial
   name: Chris Stoeckert
   orcid: 0000-0002-5714-991X
-  ror: 00b30xv10
   wikidata: Q67220657
-- affiliation: Centre for Infectious Disease Genomics and One Health, Simon Fraser University, BC
+- affiliation:
+    name: Centre for Infectious Disease Genomics and One Health, Simon Fraser University,
+      BC
+    ror: 0213rcc28
   country: Canada
   github: ddooley
   groups:
   - outreach
   name: Damion Dooley
   orcid: 0000-0002-8844-9165
-  ror: 0213rcc28
   wikidata: Q91868864
-- affiliation: Georgetown University Medical Center, Washington DC
+- affiliation:
+    name: Georgetown University Medical Center, Washington DC
+    ror: 00hjz7x27
   country: USA
   github: nataled
   groups:
   - editorial
   link: http://pir.georgetown.edu/pirwww/aboutpir/natalebio.shtml
   name: Darren Natale
   orcid: 0000-0001-5809-9523
-  ror: 00hjz7x27
   wikidata: Q99552327
-- affiliation: EMBL-EBI, Cambridge
+- affiliation:
+    name: EMBL-EBI, Cambridge
+    ror: 02catss52
   country: UK
   github: dosumis
   groups:
   - outreach
   name: David Osumi-Sutherland
   orcid: 0000-0002-7073-9172
-  ror: 02catss52
   wikidata: Q54303418
-- affiliation: SIB Swiss Institute of Bioinformatics, Basel
+- affiliation:
+    name: SIB Swiss Institute of Bioinformatics, Basel
+    ror: 002n09z45
   country: Switzerland
   github: deepakunni3
   groups:
-    - technical
+  - technical
   name: Deepak R. Unni
   orcid: 0000-0002-3583-7340
-  ror: 002n09z45
   wikidata: Q59690671
-- affiliation: Kansas State University, Manhattan, KS
+- affiliation:
+    name: Kansas State University, Manhattan, KS
+    ror: 05p1j8758
   country: USA
   github: handemcginty
   groups:
   - editorial
   - outreach
-  name: Hande Küçük McGinty
+  name: "Hande K\xFC\xE7\xFCk McGinty"
```

Comment:
This change does not look good.

Reply:
oops i forgot to fix the ascii. fixing momentarily

Reply:
Fixed in 77e53269